docs/platform/howto/update-tax-status.rst

```diff
@@ -0,0 +1,17 @@
+Update your tax status
+======================
+
+You can add your VAT ID in the billing information in the Aiven Console:
+
+1. Click **Billing** and select the billing group that you want to update.
+2. Click **Billing information**.
+3. In the **Billing Details** section, click **Edit**.
+4. Enter your VAT ID and click **Save**. 
+
+.. important::
+
+    Please be aware that you can set up one billing profile per project or create a Billing Group to group costs and consolidate invoices by different business units.
+    For further information please refer to :doc:`Billing Groups </docs/platform/concepts/billing-groups>`. 
```

Comment:
You can use the `seealso` directive for this one. In any case, I'd remove "please" and use a comma before "refer".